packages/flame_test/lib/flame_test.dart

```diff
@@ -1,3 +1,4 @@
+export 'src/close_to_vector.dart';
 export 'src/expect_double.dart';
 export 'src/expect_vector2.dart';
```

Comment:
why do we need close_to_vector and expect_vector2 ? I think we should have only one of them

Reply:
I created an issue for that: #1273